pronto_mia/lib/ui/views/deployment_plan/overview/deployment_plan_overview_viewmodel.dart

```diff
@@ -57,24 +60,31 @@ class DeploymentPlanOverviewViewModel
     );
   }
 
-  Future<void> createDeploymentPlan() async {

```

Comment:
Why can this be removed?

Reply:
This was removed because all actions in the gui now call `editDeploymentPlan()` either with a deploymentPlan or without,
which makes sense since the view that is opened is always `DeploymentPlanEditView`.

A side effect of this is that the control flow responsible for deciding how the `DeploymentPlanEditView` is opened (dialog or as a new view) and for reloading the deployment plan list is implemented in one place.